<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessApproval</class>
 <widget class="QWidget" name="AccessApproval">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Access Approval</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Client Mac</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Client IP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Approve</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>751</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_UserFullName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reject</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
